src/main/java/org/ladocuploader/app/preparers/HouseholdResourcesPreparer.java

```diff
@@ -0,0 +1,60 @@
+package org.ladocuploader.app.preparers;
+
+import formflow.library.data.Submission;
+import formflow.library.pdf.PdfMap;
+import formflow.library.pdf.SingleField;
+import formflow.library.pdf.SubmissionField;
+import formflow.library.pdf.SubmissionFieldPreparer;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.emptyList;
+
+@Component
+@Slf4j
+public class HouseholdResourcesPreparer implements SubmissionFieldPreparer {
+  private static final String AMOUNT_PREFIX = "moneyOnHandValue_wildcard_";
+  private static final String OWNER_PREFIX = "moneyOnHandOwner_wildcard_";
+  private static final List<String> RESOURCES = new ArrayList<>();
+
+  static {
+    RESOURCES.add("Checking account");
+    RESOURCES.add("Savings account");
+    RESOURCES.add("Joint account");
+    RESOURCES.add("Bonds");
+    RESOURCES.add("Cash on hand");
+    RESOURCES.add("Certificate of Deposit (CD)");
+    RESOURCES.add("Money Market Account");
+    RESOURCES.add("Mutual funds");
+    RESOURCES.add("Savings bond");
+    RESOURCES.add("Stocks");
+  }
+
+  @Override
+  public Map<String, SubmissionField> prepareSubmissionFields(Submission submission, PdfMap pdfMap) {
+    Map<String, SubmissionField> results = new HashMap<>();
+
+    var types = (List) submission.getInputData().getOrDefault("moneyOnHandTypes[]", emptyList());
+    if (!types.isEmpty()) {
+      List<String> sortedResources = RESOURCES.stream().sorted().toList();
```

Comment:
I wonder if you want to do this in the static section above, with the idea of doing it once and not per call to this?

Reply:
Or, I'm pretty sure the ArrayList will keep the order if you add them in sort order.

Reply:
Oh yeah thats a good point - the sorting is just for testing purposes. 

A lot of these preparers with "type, amount, freq" fields are really similar and I was doing a lot of c/p